test/config/integration/squash_filter.yaml

```diff
@@ -0,0 +1,61 @@
+static_resources:
```

Comment:
Please use the new style of integration test
https://github.com/envoyproxy/envoy/blob/master/test/integration/README.md

You should be able to define the yaml for your filter inline with your test and then use
config_helper_.addFilter(ConfigHelper::SQUASH_FILTER);